test/README.md

```diff
@@ -116,84 +104,76 @@ Complete Tasks 1 *Configure Environment* and 2 *Compile tests*, described in gen
   * component specific tests (WIP)
   * against an IBM Java8 SDK
```

Comment:
`IBM Java8` - this is arguable correct as OpenJ9 doesn't support Java 8 (yet) but the `IBM Java 9 SDK` should be changed to `OpenJ9 Java 9 SDK`.

Other than this last change, I think this is good to merge once modified.

Reply:
changeset is updated